test/e2e/flowaggregator_test.go

```diff
@@ -1152,6 +1160,15 @@ func checkEgressInfoClickHouse(t *testing.T, record *ClickHouseFullRow, egressNa
 	assert.Equal(t, egressIP, record.EgressIP, "Record does not have correct egressIP")
 }
 
+func checkL7VisibilityData(t *testing.T, record, l7ProtocolName string) {
+	assert.Containsf(t, record, fmt.Sprintf("l7ProtocolName: %s", l7ProtocolName), "Record does not have correct Layer 7 protocol Name")
+}
+
+func checkL7VisibilityDataClickHouse(t *testing.T, record *ClickHouseFullRow, l7ProtocolName string) {
+	assert.Containsf(t, record.L7ProtocolName, l7ProtocolName, "Record does not have correct Layer 7 protocol Name")
```

Comment:
why `Contains` and not `Equal`?

Reply:
That is because we are concerned only about the l7 related pod label.

Reply:
Sorry, still not clear to me. What is a label here? I would expect `record.L7ProtocolName == record.L7ProtocolName` to hold true.

Reply:
apologies on my end, I misunderstood it for a different assert statement. changing it to equal